ethers-solc/src/compile/mod.rs

```diff
@@ -799,6 +801,17 @@ mod tests {
         assert_eq!(res.solc, expected);
     }
 
+    #[test]
+    #[cfg(feature = "svm-solc")]
+    fn can_install_solc_in_tokio_rt() {
+        let ver = "0.8.6";
+        let version = Version::from_str(ver).unwrap();
+        let result = tokio::runtime::Runtime::new()
+            .unwrap()
+            .block_on(async { Solc::blocking_install(&version) });
```

Comment:
great!

can we also add some integration tests that pin a specific solc version and make the tests `#[tokio::test] async`?